lib/ansible/cli/vault.py

```diff
@@ -99,6 +99,9 @@ def init_parser(self):
         enc_str_parser.add_argument('-p', '--prompt', dest='encrypt_string_prompt',
                                     action='store_true',
                                     help="Prompt for the string to encrypt")
+        enc_str_parser.add_argument('-i', '--insecure', dest='show_string_to_encrypt',
```

Comment:
not sure we want this type of option, we could always double prompt as we do for passwords

Reply:
If we double prompt that would potentially break any existing automated scripts. Let me know if you want me to take this out, it was really just so we could mimic what used to happen.

Reply:
@bcoca This came up again. Any thoughts on double prompting vs the --insecure flag I proposed?